macOS/DuckDuckGo/VisualRefresh/VisualStyleConfigurable.swift

```diff
@@ -41,18 +111,30 @@ final class VisualStyleManager: VisualStyleConfigurable {
 
     private var cancellables: Set<AnyCancellable> = []
 
-    private var isEnabled: Bool {
-        featureFlagger.isFeatureOn(.visualRefresh)
-    }
-
     init(featureFlagger: FeatureFlagger) {
         self.featureFlagger = featureFlagger
 
         subscribeToLocalOverride()
     }
 
-    var toolbarHeight: CGFloat {
-        currentStyle.toolbarHeight
+    // MARK: - VisualStyleConfigurable implementation
+
+    func addressBarHeight(for type: AddressBarSizeClass) -> CGFloat {
+        return currentStyle.addressBarHeight(for: type)
+    }
+
+    func addressBarTopPaddig(for type: AddressBarSizeClass) -> CGFloat {
+        return currentStyle.addressBarTopPaddig(for: type)
+    }
+
+    func addressBarBottomPaddig(for type: AddressBarSizeClass) -> CGFloat {
+        return currentStyle.addressBarBottomPaddig(for: type)
+    }
+
+    // MARK: - Private properties
+
+    private var isEnabled: Bool {
```

Comment:
If I can be honest with you, `isEnabled` sounds more cryptic than `featureFlagger.isFeatureOn(.visualRefresh)` :D because it’s hard to figure out for a reader what actually is enabled. I’d personally remove it and use the feature flagger directly inside `currentStyle`.

Reply:
Honesty is what I want in these reviews :). Removed the `isEnabled` I will use isFeatureOn instead